.github/workflows/anaconda_upload.yml

```diff
@@ -6,10 +6,15 @@ on:
       - main
     tags:
       - '*'
+  workflow_run:
+    workflows: ["Test"]
+    types:
+      - completed
 
 jobs:
   anaconda_org_build_and_upload:
     name: anaconda.org upload py${{ matrix.python-version }}
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.head_branch == 'main' }}
```

Comment:
This condition would make it such that the github workflow is only triggered on `main` branch and only on the `sucess` state of the `Test` workflow.

Reply:
I reviewed this via `esc/pixie`:

Basically, I merged this PR to `esc/pixie@main` and then deliberately introduced a breaking change.

The result was that the upload action was skipped:

https://github.com/esc/pixie/actions/runs/9284604241